### ISSUE TITLE
Photo Consent

### ISSUE THREAD

**Issue by rugulous** [BritishYouthBandAssociation/OrgAdmin#231]
We need to record if a band has consented to photography/videography or not.

We also need to keep a log of any changes to this - for example if consent is granted, we take a load of photos, THEN consent is revoked, it is a lot different to taking photos without consent.

If a band has consented, we also need an optional "notes" field for recording details of any members that cannot be photographed even if the rest can.

These fields should be added to the `Organisation` profile page

**Comment by rugulous** [comment]
This will need adding to the bit that calls the WooCommerce API for details about current memberships too

**Comment by rugulous** [comment]
This is actually rather important so bringing forward

**Comment by rugulous** [comment]
What I'm going for is an auditable history, so we will know when we did/didn't have consent.

The Organisation profile screen will display the **CURRENT** status, with a link to view the history. Users will also be able to change the status from this screen, but will be prompted to give a reason for changes.

The history will be a longer page that will allow us to see exactly when the status was changed (and by who). It will also let us see any comments that have been added.

I envision 3 statuses:
1 - No photos at all (photography is blanket banned)
2 - Yes, but... (most people in the band consent, besides xyz - the user MUST enter the requirements)
3 - Yes (photography is blanket allowed)
